python/sglang/srt/layers/quantization/unquant.py

```diff
@@ -361,6 +361,9 @@ def forward_cpu(
                 correction_bias=correction_bias,
                 routed_scaling_factor=routed_scaling_factor,
             )
+            x, topk_weights = apply_topk_weights_cpu(
+                apply_router_weight_on_input, topk_weights, x
+            )
 
             # TODO: support apply_router_weight_on_input in the fused_experts_cpu kernel
```

Comment:
should we remove this TODO?

Reply:
Thanks for that, have removed this notes.